src/main/java/calculator/domain/Expression.java

```diff
@@ -7,6 +7,10 @@
 import java.util.stream.IntStream;
 
 public class Expression {
+    public static int ZERO = 0;
+    public static int ODD_RESULT = 1;
+    public static int EVEN_CONDITION = 2;
+    public static String[] OPERATORS = {"+", "-", "*", "/"};
```

Comment:
피드백 반영 👍 
final 키워드도 추가되면 더 좋을 것 같네요.
배열보단 List 타입으로 선언해도 괜찮을 것 같아요.
```suggestion
    public static final List<String> OPERATORS = List.of("+", "-", "*", "/");
```